backend/tests/core/api/algolia_test.py

```diff
@@ -88,3 +88,78 @@ def test_algolia_search_invalid_method(self):
 
         assert response.status_code == requests.codes.method_not_allowed
         assert response_data["error"] == "Method GET is not allowed"
+
+    @pytest.mark.parametrize(
```

Comment:
You should test validators functions -- create `tests/core/validators_test.py` and test each method with corresponding cases.

Keep some basic tests here too to make sure `algolia_search` is also covered